examples/echo-api-key-simple.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   identity:
     - name: friends # user defined
       credentials:
-        in: custom_header # options include: custom_header, authorization_header and query
+        in: custom_header # options include: custom_header, authorization_header, cookie and query
```

Comment:
We need to document this in the OpenAPI spec for the CRD. Let's make it in another PR because there are other fields and options missing in there as well (e.g. the whole part on "json" authorization).